core-ui/src/components/Clusters/views/EditCluster/EditCluster.js

```diff
@@ -74,9 +74,12 @@ function EditClusterComponent({
     findInitialValue('oidc-issuer-url'),
   );
   const [clientId, setClientId] = useState(findInitialValue('oidc-client-id'));
+  const [clientSecret, setClientSecret] = useState(
+    findInitialValue('oidc-client-id'),
+  );
```

Comment:
I think there should be `oidc-client-secret`.
```suggestion
  const [clientSecret, setClientSecret] = useState(
    findInitialValue('oidc-client-secret'),
  );
```

Reply:
omg yes of course, good catch